src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,18 +19,35 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
```

Comment:
Remember to close the `SessionFactory` object after you're done using it to free up resources. This can be done in a `finally` block or using a try-with-resources statement if you refactor `getSessionFactory` to implement `AutoCloseable`.